example/server/storage/storage.go

```diff
@@ -424,6 +452,10 @@ func (s *Storage) SetIntrospectionFromToken(ctx context.Context, introspection o
 // GetPrivateClaimsFromScopes implements the op.Storage interface
 // it will be called for the creation of a JWT access token to assert claims for custom scopes
 func (s *Storage) GetPrivateClaimsFromScopes(ctx context.Context, userID, clientID string, scopes []string) (claims map[string]interface{}, err error) {
+	return s.getPrivateClaimsFromScopes(ctx, userID, clientID, scopes)
+}
+
+func (s *Storage) getPrivateClaimsFromScopes(ctx context.Context, userID, clientID string, scopes []string) (claims map[string]interface{}, err error) {
```

Comment:
Unnecesary change

Reply:
The logic moved to private function to reuse it in 

https://github.com/zitadel/oidc/blob/69e8cb434da870550564b9bf7526c4bd03182d19/example/server/storage/storage.go#L640

Reply:
Exported functions can also be reused in the same package. Since the function signature remains the same, please revert this change and just reuse the exported function.